### PR TITLE
Merge Range() into Row() call.

### LINKED ISSUE #1757: Move Range (time) functionality to Row call and support UNIX timestamps
### Description

In a previous ticket https://github.com/pilosa/pilosa/issues/1318 it was proposed to move `Range(<FIELD>=<ROW>, <DATE_TIME>, <DATE_TIME>)` call to `Row(<FIELD>=<ROW>, start=<TIMESTAMP>, end=<TIMESTAMP>)` call but that was never realized.

Overloading the `Row` call with the time range has the following advantages:
* One less call name to remember.
* `Row` becomes more symmetrical with `Set` which can set the timestamp of the column.
* It becomes possible to leave one of `start` (to get all columns after the start) or `end` (to get all columns up to the end). If both `start` and `end` are left out, `Row` calls behaves as usual.

On top of that, it would be desirable to be able to specify UNIX timestamps in the time range. Currently used timestamp format is fine for manually running queries, but it is more common and easier to specify UNIX timestamp in code.

### Success criteria (What criteria will consider this ticket closable?)

* `Row` call supports `start` and `end` parameters to specify the time range.
* `start` and `end` parameters supports UNIX timestamps.
* `Range` (time) call is marked deprecated.

### PR DESCRIPTION
**Pull request by benbjohnson**
## Overview

This commit refactors the `Range()` call and merges its functionality into the `Row()` call.

Fixes #1757

## Pull request checklist

- [x] I have read the [contributing guide](https://github.com/pilosa/pilosa/blob/master/CONTRIBUTING.md).
- [x] I have agreed to the [Contributor License Agreement](https://cla-assistant.io/pilosa/pilosa).
- [x] I have updated the [documentation](https://github.com/pilosa/pilosa/tree/master/docs).
- [x] I have resolved any merge conflicts.
- [x] I have included tests that cover my changes.
- [x] All new and existing tests pass.

## Code review checklist
This is the checklist that the reviewer will follow while reviewing your pull request. You do not need to do anything with this checklist, but be aware of what the reviewer will be looking for.

- [ ] Ensure that any changes to external docs have been included in this pull request.
- [ ] If the changes require that minor/major versions need to be updated, tag the PR appropriately.
- [ ] Ensure the new code is [properly commented](https://github.com/golang/go/wiki/CodeReviewComments#doc-comments) and follows [Idiomatic Go](https://dmitri.shuralyov.com/idiomatic-go).
- [ ] Check that tests have been written and that they cover the new functionality.
- [ ] Run tests and ensure they pass.
- [ ] Build and run the code, performing any applicable integration testing.

### CODE REVIEW

**Comment by benbjohnson** [comment]
@travisturner @jaffee I fixed up the issues with the conditionals, did some more testing, and updated the documentation. This pull request is ready for review.

**Comment by jaffee** [comment]
I haven't done a deep review, but this appears to remove `Range` entirely. My understanding was that we were going to leave `Range` alone, but remove it from documentation, add a deprecation warning if it's used, and enable `Row` to support a superset of its functionality. 

As is, this would require a 2.0 release, correct?

**Comment by benbjohnson** [comment]
@jaffee Ah, shoot, sorry about that. You're right. I'll add the `Range()` support back in. 👍

**Comment by benbjohnson** [comment]
@jaffee I updated the PR to continue to support `Range()` and add a deprecation warning. (06a1247)

File: executor.go [hunk 1]
```diff
@@ -1292,9 +1250,17 @@ func (e *executor) executeRangeShard(ctx context.Context, index string, c *pql.C
 		return &Row{}, nil
 	}
 
+	// Set maximum "to" value if only "from" is set. We don't need to worry
+	// about setting the minimum "from" since it is the zero value if omitted.
+	if toTime.IsZero() {
+		// This is the maximum comparible time.Time value.
```
Comment (jaffee) [inline executor.go]: s/comparible/comparable/ I think

**Comment by jaffee** [comment]
approved and kicked... permissions on CircleCI are eluding me at the moment - I *think* you might need to link Circle to github and "follow" the pilosa workflow, but honestly, I'm not clear. @codysoyland might know better.

**Comment by codysoyland** [comment]
I'm confused by the CircleCI permissions as well. Ben is not listed as a person I can invite to our team in CircleCI, but it is supposed to list everybody who is in the Github Org. I would try what Jaffee said and create and link your Github account.

**Comment by jaffee** [comment]
glad I'm not the only one who's scratching their head over that

**Pull request merged**